content/en/includes/task-tutorial-prereqs-node-upgrade.md

```diff
@@ -0,0 +1,8 @@
+You need to have a Kubernetes cluster, shell access to all the nodes, and the kubectl command-line tool must
+be configured to communicate with your cluster. It is recommended to run this tutorial on a cluster with at least two nodes that are not acting as control plane hosts. If you do not already have a
+cluster, you can create one by using
+[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
+or you can use one of these Kubernetes playgrounds:
+
+* [Killercoda](https://killercoda.com/playgrounds/scenario/kubernetes)
+* [Play with Kubernetes](https://labs.play-with-k8s.com/)
```

Comment:
You never need to upgrade nodes when you don't have a cluster, so we can narrow this advice.

Reply:
Would this work? ...

You need to have shell access to all the nodes, and the kubectl command-line tool must be configured to communicate with your cluster. It is recommended to run this tutorial on a cluster with at least two nodes that are not acting as control plane hosts. 

Familiarize yourself with the [process for upgrading the rest of your kubeadm cluster](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade). You will want to upgrade the control plane nodes before upgrading your Linux Worker nodes.

Reply:
Sure, give that a try.